Dockerfile

```diff
@@ -1,7 +1,12 @@
-FROM node:14-alpine as build
+FROM node:14-alpine as base
 WORKDIR /usr/app
 RUN apk update && apk add --no-cache git g++ make python && rm -rf /var/cache/apk/*
 
+FROM base as development
+RUN apk add --no-cache bash
```

Comment:
why not add this to L3?

Reply:
whats L3?

Reply:
Line 3 of this file :)

Reply:
makes sense, could shift there as final "production" gets the copies from "build" target, but only catch is a little bit extra time it might take to add bash package for production build. so in my opinion this is just fine unless you strongly feel otherwise.

Reply:
From what I see, production image is fresh nodejs-alpine image so it won't have bash installed. I'm asking because this new apk add probably adds new docker layer which probably makes image size larger. Do you have docker image sizes before and after?

Reply:
yes, final production image gets copy from build, i think it shouldn't affect as in my production target build the development target was not even executed, so i think it shouldnt be adding any extra layers. however will confim on image sizes.
![image](https://user-images.githubusercontent.com/76567250/111956478-d4bfa880-8b10-11eb-9132-f6084945ab94.png)

Reply:
@mpetrunic yep negligible diff ~1MB, no development target layers included